app/views/admin/videos/new.html.haml

```diff
@@ -0,0 +1,22 @@
+%section.admin_add_video
+  .container
+    .row
+      .col-md-10.col-md-offset-1
+        = bootstrap_form_for [:admin, @video], class: "form-horizontal" do |f|
+          %ul.nav.nav-tabs
+            %li
+              %a(href="/ui/admin_views_payments") Recent Payments
+            %li.active
+              %a(href="") Add a New Video
+          %br
+          %fieldset
+            = f.text_field :title, class: "span3"
```

Comment:
`span*` are classes from Bootstrap v2. You need to use `col-*-*` now. You can see more details in the bootstrap docs.